crawl-ref/source/main.cc

```diff
@@ -2921,8 +2921,13 @@ static void _move_player(coord_def move)
             // Don't choose a random location to try to attack into - allows
             // abuse, since trying to move (not attack) takes no time, and
             // shouldn't. Just force confused trees to use ctrl.
+#ifdef USE_TILE_WEB
             mpr("You cannot move. (Use ctrl+direction or * direction to "
```

Comment:
Can someone confirm for me that #ifdef USE_TILE_WEB returns true only for WebTiles games and not online console games? aidanh was worried that it might accept all online games.

Reply:
It does accept online console games, thanks |amethyst, but I'm not really sure whether this is an issue at all.